metals/src/main/scala/scala/meta/internal/metals/DefinitionProvider.scala

```diff
@@ -44,7 +45,8 @@ final class DefinitionProvider(
     icons: Icons,
     statusBar: StatusBar,
     warnings: Warnings,
-    compilers: () => Compilers
+    compilers: () => Compilers,
+    remote: RemoteLanguageServer
```

Comment:
Maybe `Option[RemoteLanguageServer]` ? It will be more clear that it might not be available.